include/libopencm3/stm32/h7/rcc.h

```diff
@@ -182,29 +182,37 @@ LGPL License Terms @ref lgpl_license
 /** @defgroup rcc_d1cfgr_values RCC_D1CFGR Values
  * @ingroup rcc_registers
  * @{*/
-#define RCC_D1CFGR_D1CPRE_BYP       0x0
-#define RCC_D1CFGR_D1CPRE_DIV2      0x8
-#define RCC_D1CFGR_D1CPRE_DIV4      0x9
-#define RCC_D1CFGR_D1CPRE_DIV8      0xA
-#define RCC_D1CFGR_D1CPRE_DIV16     0xB
-#define RCC_D1CFGR_D1CPRE_DIV64     0xC
-#define RCC_D1CFGR_D1CPRE_DIV128    0xD
-#define RCC_D1CFGR_D1CPRE_DIV256    0xE
-#define RCC_D1CFGR_D1CPRE_DIV512    0xF
-#define RCC_D1CFGR_D1PPRE_BYP       0x0
-#define RCC_D1CFGR_D1PPRE_DIV2      0x4
-#define RCC_D1CFGR_D1PPRE_DIV4      0x5
-#define RCC_D1CFGR_D1PPRE_DIV8      0x6
-#define RCC_D1CFGR_D1PPRE_DIV16     0x7
-#define RCC_D1CFGR_D1HPRE_BYP       0x0
-#define RCC_D1CFGR_D1HPRE_DIV2      0x8
-#define RCC_D1CFGR_D1HPRE_DIV4      0x9
-#define RCC_D1CFGR_D1HPRE_DIV8      0xA
-#define RCC_D1CFGR_D1HPRE_DIV16     0xB
-#define RCC_D1CFGR_D1HPRE_DIV64     0xC
-#define RCC_D1CFGR_D1HPRE_DIV128    0xD
-#define RCC_D1CFGR_D1HPRE_DIV256    0xE
-#define RCC_D1CFGR_D1HPRE_DIV512    0xF
+enum rcc_core_pre {
```

Comment:
I think the enum naming should stick with the existing naming convention, something like rcc_d1cfgr_[your-free-text-here]  things like "rcc_core_" are going to start colliding or being non-visible quite quickly if they're not standardized.